### ISSUE TITLE
Mixed spaces and tabs

### ISSUE THREAD

**Issue by szkrd** [douglascrockford/JSLint#119]
Mixed spaces and tabs are triggered when spaces are followed by tabs, but not the other way around. Is this because of the jsdoc one space after tab convention? I have forked a patch along with some other modifications (trailing whitespaces, consecutive empty lines)  [here](https://github.com/rosamez/JSLint/commit/ca4c39a62dedb2a5f58a5c19394a855bd2c7f0c8#jslint.js) - but if it is not the case, please let me know. Thank you.

Unfortunately github replaces the hard tabs, so:

```
(function () {
$$$$"use strict";
TTTTvar x = 1,
TTTT$$$$y = 2,
$$$$TTTTz = 3;
$$$$x = x + y + z;
}());
```
